roffit

```diff
@@ -152,7 +152,7 @@ p.roffit {
  text-align: center;
  font-size: 80%;
 }
-</STYLE>
+</style>
```

Comment:
These are off-topic changes. I think you should do them separately.

Reply:
Ok but if @1268 wants to do them he should. Will give it a week or so and if not resolved, then will re-open prior pull \#40.